src/components/AddContainerDialogComponent.vue

```diff
@@ -33,28 +47,50 @@ import type { Ref } from "vue";
 import type { UserResponse } from "@sudosos/sudosos-client";
 import { useAuthStore } from "@sudosos/sudosos-frontend-common";
 import { useContainerStore } from "@/stores/container.store";
+import * as yup from 'yup';
```

Comment:
The ordering seems to be quite random?

Prefer
imports
emits
refs
stores

lifecycle hooks if possible

remaining code

Reply:
Should be mostly according to this now.